ssl/ssltest.c

```diff
@@ -470,10 +470,17 @@ static int cb_server_alpn(SSL *s, const unsigned char **out,
 static int verify_alpn(SSL *client, SSL *server)
 {
     const unsigned char *client_proto, *server_proto;
+    const unsigned char null_protocol[1] = {0};
```

Comment:
Not protected by PENDANTIC and I cannot find an equivalent change in master

Reply:
In master the code has been changed to:
```
   if (client_proto != NULL &&
        memcmp(client_proto, server_proto, client_proto_len) != 0) {
        BIO_printf(bio_stdout, "ALPN selected protocols differ!\n");
        goto err;
    }
```